### PR TITLE
Add ginkgo package

### PR DESCRIPTION
**Pull request by gfcas**
This adds the ginkgo library.

### CODE REVIEW

File: candi.cfg [hunk 1]
```diff
@@ -64,6 +64,7 @@ PACKAGES="load:dealii-prepare"
 #PACKAGES="${PACKAGES} once:adolc"
 #PACKAGES="${PACKAGES} once:arpack-ng"
 #PACKAGES="${PACKAGES} once:assimp"
+# PACKAGES="${PACKAGES} once:ginkgo"
```
Comment (tjhei) [inline candi.cfg:67]:
```suggestion
#PACKAGES="${PACKAGES} once:ginkgo"
```

**Review by tjhei — COMMENTED** [review]
Looks good! 👍

**Review by koecher — APPROVED** [review]

**Comment by koecher** [comment]
@gfcas I think this is good to merge, right?

**Comment by gfcas** [comment]
Worked on my laptop at least ;-)

**Pull request merged**